run_testlist.py

```diff
@@ -15,106 +15,220 @@
-               'nClient' : [ 1,  1,  1,  1,  1,  1,   1,   1,   1],
-               'timeout' : [15, 15, 15, 15, 15, 15,  15,  15,  15], #timeout in minutes
-               'ppc'     : 1,
-               'inflight': 16,
-               'enabled' : 0
-             }
-            ]
-
-ior_testlist = [ { 'testcase': 'ioreasy_1to4',
-               'nServer' : [ 2,  4,  8, 16],
-               'nClient' : [ 8, 16, 32, 64],
-               'timeout' : [15, 15, 15, 15], #timeout in minutes
-               'ppc'     : 32,
-               'pool_sz' : '85G',
-               'xfer_sz' : '1M',
-               'blk_sz'  : '1G',
-               'enabled' : 0
-             },
-             { 'testcase': 'ioreasy_c16',
-               'nServer' : [ 2,  4,  8, 16],
-               'nClient' : [16, 16, 16, 16],
-               'timeout' : [15, 15, 15, 15], #timeout in minutes
-               'ppc'     : 32,
-               'pool_sz' : '85G',
-               'xfer_sz' : '1M',
-               'blk_sz'  : '1G',
-               'enabled' : 0
-             }
-            ]
+slf_testlist = [{'testcase': 'st_1tomany_cli2srv_inf1',
+                 'nServer': [2, 4, 8, 16, 32, 64, 128, 256, 512],
+                 'nClient': [1, 1, 1, 1, 1, 1, 1, 1, 1],
+                 # timeout in minutes
+                 'timeout': [15, 15, 15, 15, 15, 15, 15, 15, 15],
+                 'ppc': 1,
+                 'inflight': 1,
+                 'enabled': 0
+                 },
+                {'testcase': 'st_1tomany_cli2srv_inf16',
+                 'nServer': [2, 4, 8, 16, 32, 64, 128, 256, 512],
+                 'nClient': [1, 1, 1, 1, 1, 1, 1, 1, 1],
+                 # timeout in minutes
+                 'timeout': [15, 15, 15, 15, 15, 15, 15, 15, 15],
+                 'ppc': 1,
+                 'inflight': 16,
+                 'enabled': 0
+                 }
+                ]
+
+ior_testlist = [{'testcase': 'ioreasy_1to4',
+                 'nServer': [2, 4, 8, 16],
+                 'nClient': [8, 16, 32, 64],
+                 # timeout in minutes
+                 'timeout': [15, 15, 15, 15],
+                 'ppc': 32,
+                 'pool_sz': '85G',
+                 'xfer_sz': '1M',
+                 'blk_sz': '1G',
+                 'enabled': 0
+                 },
+                {'testcase': 'ioreasy_c16',
+                 'nServer': [2, 4, 8, 16],
+                 'nClient': [16, 16, 16, 16],
+                 # timeout in minutes
+                 'timeout': [15, 15, 15, 15],
+                 'ppc': 32,
+                 'pool_sz': '85G',
+                 'xfer_sz': '1M',
+                 'blk_sz': '1G',
+                 'enabled': 0
+                 },
+                {'testcase': 'iorhard_1to4',
+                 'nServer': [2, 4, 8, 16, 32, 64, 128, 256],
+                 'nClient': [8, 16, 32, 64, 128, 256, 512, 1024],
+                 # timeout in minutes
+                 'timeout': [15, 15, 15, 15, 120, 120, 120, 120],
```

Comment:
I don't know how long it takes to run 32-256 servers  but we might want to adjust the timeout here at some point so it doesnt burn time on Frontera if they are expected to take much less time to run

Reply:
I don’t know either :(  Unfortunately, my bigger runs crashed, or they didn’t run at all because of I don’t have access to the large queue.
As soon as I have an average execution time, I will adjust these values. On the bright side. We are not charged by the time we requested but the time we actually used.
[From the user guide](https://frontera-portal.tacc.utexas.edu/user-guide/running/) :)
"The system charges only for the resources you actually use, not those you request"

I totally agree that these timeouts must be adjusted.